### DISCUSSION TITLE
improve syntax for excluded folders

### DISCUSSION THREAD

**Discussion by erclu** [erclu/check-crlf#3] [Ideas]
should probably use something more refined than the space-separated strings currently implemented

**Comment by jmckenna** [comment]
I also would like to make a feature request to exclude filetypes by extension (such as *.pdf).   Should I file this as a separate ticket?

**Comment by erclu** [comment]
oh yeah I agree that is an expected functionality. I'll try and rework how the filtering is done, ideally i'd do something that works the same as gitignore but only through the GitHub Actions arguments since requiring a file would probably be an overkill
